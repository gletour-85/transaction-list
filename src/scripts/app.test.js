import App from './App';
import renderer from 'react-test-renderer';

jest.mock('@material-ui/core', () => ({
  AppBar: 'AppBar',
  Container: 'Container',
  CssBaseline: 'CssBaseline',
  Slide: 'Slide',
  Toolbar: 'Toolbar',
  Typography: 'Typography',
  useScrollTrigger: jest.fn()
}));
jest.mock('./components/transactions-table', () => 'TransactionsTable');

describe('App', () => {
  it('renders correctly', () => {
    expect(renderer.create(<App />)).toMatchSnapshot();
  });
});
