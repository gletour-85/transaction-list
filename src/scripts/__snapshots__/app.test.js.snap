// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
Array [
  <CssBaseline />,
  <Slide
    appear={false}
    direction="down"
    in={true}
  >
    <AppBar
      className="page-header"
    >
      <Toolbar
        className="page-header"
      >
        <Typography
          className="text--centered"
          variant="h5"
        >
          Bench Test
        </Typography>
      </Toolbar>
    </AppBar>
  </Slide>,
  <Container
    className="main"
    component="main"
    role="main"
  >
    <TransactionsTable />
  </Container>,
]
`;
